### DISCUSSION TITLE
How to know which brand a Sponsored Brand campaign is attached to

### DISCUSSION THREAD

**Discussion by Luscha** [amzn/ads-advanced-tools-docs#141] [Q&A]
I coulnd't find any way to know which brand is being advertised using SB report v2, v3 or SB v3 apis.
None of those data sources return a `BrandId`, `BrandEntityId` or `BrandRegisteredName`.

What is the correct way to bind a brand to a sb campaign?

**Comment by elizafarley** [comment]
You are correct that brand ID is not currently available via reporting. 

There are a couple of workarounds you can try to see this information in reports:

1. Include the brand name in campaignName.
2. Create separate ad groups for each brand family you are advertising.
3. Create campaign portfolios by brand. 

You can also look into the [Brand Metrics APIs](https://advertising.amazon.com/API/docs/en-us/brand-metrics-openapi) that offer brand insights at the category (but not campaign) level.

If you have any feedback on how this works currently, please let me know so I can pass it on to the product team.
